From 05f69f367b1f1821efe34b50a423829ac93ae099 Mon Sep 17 00:00:00 2001
From: Muhammad Usama Anjum <usama.anjum@collabora.com>
Date: Wed, 15 Oct 2025 14:25:55 +0500
Subject: [PATCH 036/188] PM: hibernate: export
 pm_sleep_transition_in_progress()

Export pm_sleep_transition_in_progress() to be used by other
modules.

Signed-off-by: Muhammad Usama Anjum <usama.anjum@collabora.com>
---
 kernel/power/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/power/main.c b/kernel/power/main.c
index 5429e9f19..7f8a44470 100644
--- a/kernel/power/main.c
+++ b/kernel/power/main.c
@@ -639,6 +639,7 @@ bool pm_sleep_transition_in_progress(void)
 {
 	return pm_suspend_in_progress() || hibernation_in_progress();
 }
+EXPORT_SYMBOL_GPL(pm_sleep_transition_in_progress);
 #endif /* CONFIG_PM_SLEEP */
 
 #ifdef CONFIG_PM_SLEEP_DEBUG
-- 
2.55.0

